.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+    - 8
+script:
+    - npm run check
+    - npm test
```

Comment:
I'd like to add `codecov` as well, to be consistent with our other GitHub projects. It could look like this:

```
language: node_js
node_js:
    - 8
install:
    - npm install
    - npm install -g codecov
script:
    - npm run check
    - npm test
after_success:
    - codecov
```

Reply:
Fixed